### ISSUE TITLE
[Bug]: Bismuth Window Decoration disables all Animations

### ISSUE THREAD

**Issue by seykodev** [Bismuth-Forge/bismuth#414]
### Summary

When installing Bismuth it comes with an window decoration which removes the titlebars and adds borders around the windows to indicate the active window. While this function is really handy for me it creates a strange behavior while using it. Whenever i select the Bismuth Window Decoration, all animations in programs like Firefox, Discord or Chrome are gone. It behaves like i set the Animation Speed in KDE Settings to instant, but it is not set to that. To fix this i have to change the Animation Speed again but after rebooting the System all animations are gone again. Whenever i select a different window decoration, all animations are working fine again.

What I have tried to fix this issue:
Reinstall and downgrade the NVIDIA Drivers.
Trying Wayland instead of X11

### Steps to Reproduce

1. Enable the window decoration by going to System Settings > Appearance > Window Decorations and selecting the Bismuth decoration.
2. Make sure that the animation speed is not set to instant.
3. Reboot the system.
4. Animations in a lot of Applications like Chrome, Firefox or Discord are gone (this includes: Smooth Scrolling and the tab loading animations or tab opening animations)

### Expected behavior

1. Enable the window decoration by going to System Settings > Appearance > Window Decorations and selecting the Bismuth decoration.
2. Titlebars are removed and there is a border around the active window.
3. All other animations inside applications are still there.

### Screenshots

This is what the loading animation in Firefox looks like while I use the Window Decoration, which comes with Busmith:
![2022-08-17_13-52](https://user-images.githubusercontent.com/107946893/185112143-08f26693-9ce4-43aa-9766-30ef949e4e94.png)

### Bismuth version

3.1.2

### KDE Plasma version

5.25.4

### The platform KWin is running on

X11

**Comment by seykodev** [comment]
I found out that the Bismuth Window Decoration seems to change gtk-enable-animations to false in the GTK configuration on every boot, which causes the lack of animations. I fixed this by creating a script which sets the value back to true on every boot. Is there any reason why the Window Decoration is changing this config value?

**Comment by gnufred** [comment]
I'm pretty sure that's a "feature" and not a bug. Tilling managers like Sway/i3 don't have any animation either. I'm pretty sure that's the desired outcome.

**Comment by atagen** [comment]
Animation in tiling managers is often sought after - see, for example, the many picom forks people use with i3, or animation libraries written for awesome. Beyond the aesthetic value, it is useful for clearly demonstrating to the user what just moved where - surely a useful option to leave in?

**Comment by Nano-Ocelot** [comment]
> Animation in tiling managers is often sought after; see, for example, the many picom forks people use with i3, or animation libraries like rubato written for awesome. Beyond the aesthetic value, it is useful for clearly demonstrating to the user what just moved where - surely a useful option to leave in?

I agree. This seems like something to leave up to the user instead of having to rely on strange workarounds.

**Comment by seykodev** [comment]
I am aware that tiling window managers are not using animations out of the box. But i never experienced a tiling window manager actually modifying a GTK config to disable animations in all applications. At least whenever i run i3, applications like Firefox still got animations and smooth scrolling enabled. It is a good thing to include in Bismuth's window decorations since some people might prefer the usage without any animations but as already mentioned here, it should be something with can be easily toggled instead of being forced to the user. I mean everything works fine with the workaround i did, but i feel like that's just not really a convenient method to get animations back.
